### DISCUSSION TITLE
User's guide structure improvement

### DISCUSSION THREAD

**Discussion by ewaagaard** [xsuite/xsuite#262] [Announcements]
I believe that the overarching structure of [Xsuite user's guide](https://xsuite.readthedocs.io/en/latest/usersguide.html) could be further enhanced. Now we start having too many articles, many of which fall into the same subdomain. 

For instance, I would suggest that 
- [Installation](https://xsuite.readthedocs.io/en/latest/installation.html) and [Getting started: single-particle tracking](https://xsuite.readthedocs.io/en/latest/singlepart.html) remain as their own categories
- [Save an Xtrack Line to a json file](https://xsuite.readthedocs.io/en/latest/line2json.html) and [Data management in Xsuite](https://xsuite.readthedocs.io/en/latest/xsuite_data_management.html) could remain separate, but under a new category called for instance "Saving and loading objects"
- [Deferred expressions from MAD-X](https://xsuite.readthedocs.io/en/latest/deferred_expressions.html) and [Interface to PyHEADTAIL](https://xsuite.readthedocs.io/en/latest/pyhtinterface.html) could maybe be included under a new category called "Working with other simulation codes" or similar
- [Particles monitors](https://xsuite.readthedocs.io/en/latest/particles_monitor.html) and [Working with Particles objects](https://xsuite.readthedocs.io/en/latest/particlesmanip.html) could maybe enter a new category "Tracking particles"
- Everything related to 4D and 6D Twiss, and [Match twiss parameters](https://xsuite.readthedocs.io/en/latest/match_twiss_params.html), could maybe be grouped together under "Transverse dynamics" or even "Twiss"
- [Freeze longitudinal coordinates](https://xsuite.readthedocs.io/en/latest/freeze_longitudinal.html) and some others could maybe enter a new category "Longitudinal dynamics"
- [Combined CPU - GPU simulations](https://xsuite.readthedocs.io/en/latest/combined_cpu_gpu.html) and [Optimize line and tracker for multi-turn tracking speed](https://xsuite.readthedocs.io/en/latest/optimize_for_tracking.html) could maybe go into the new category "Speeding up simulations" 
- [Space charge](https://xsuite.readthedocs.io/en/latest/spacechargeauto.html) and eventually IBS could go under "Collective effects"

These new categories are only suggested names, there could also be a more logical structure. But I think that for a new user new categories of this type could be very helpful.

**Comment by giadarol** [comment]
Hi Elias, thanks for the feedback.
I think for now a flatter structure helps the user to find things and notice new additions.
Definitely to be reviewed at a later stage when the guide will grow further.
